jkube-kit/build/api/pom.xml

```diff
@@ -31,6 +31,14 @@
 
   <dependencies>
 
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <version>5.7.0</version>
+        <scope>test</scope>
+    </dependency>
```

Comment:
why is this needed?

Reply:
The import for assertThrows from "junit-jpuiter-api" was requiring specification of that dependency in pom.xml.

Since you have already suggested the changes, ill use the Assert.assertThrows import